integration/cri-o/cri-o.sh

```diff
@@ -15,6 +15,16 @@ source /etc/os-release
 crio_repository="github.com/kubernetes-incubator/cri-o"
 crio_repository_path="$GOPATH/src/${crio_repository}"
 
+# devicemapper device and options
+export LVM_DEVICE=${LVM_DEVICE:-/dev/vdb}
+DM_STORAGE_OPTIONS="--storage-driver devicemapper --storage-opt dm.directlvm_device=${LVM_DEVICE}
```

Comment:
Nit: I'd prefer that we have each option on a separate line (with a backslash) as that would be easier to read and make diffs smaller when we need to change the options.